WalletWasabi/Services/WalletService.cs

```diff
@@ -253,6 +253,22 @@ private async void IndexDownloader_NewFilterAsync(object sender, FilterModel fil
 			}
 		}
 
+		public static void ResyncWallet(string walletName, string DataDir, Network network)
+		{
+			string BlocksFolderPath = Path.Combine(DataDir, "Blocks", network.ToString());
+			string TransactionsFolderPath = Path.Combine(DataDir, "Transactions", network.ToString());
+			if (Directory.Exists(BlocksFolderPath))
+			{
+				Directory.Delete(BlocksFolderPath, true);
+				Directory.CreateDirectory(BlocksFolderPath);
+			}
+			string TransactionsFilePath = Path.Combine(TransactionsFolderPath, $"{walletName}Transactions.json");
+			if (File.Exists(TransactionsFilePath))
+			{
+				File.Delete(TransactionsFilePath);
+			}
```

Comment:
`DataDir`,  `BlocksFolderPath`, `TransactionsFolderPath` and `TransactionsFilePath` should all be `camelCase`.

Reply:
Once #1920 gets merged it will be easier to detect this.